middleware/gzip/gzip_test.go

```diff
@@ -0,0 +1,100 @@
+package gzip
+
+import (
+	"fmt"
+	"net/http"
+	"net/http/httptest"
+	"testing"
+
+	"github.com/mholt/caddy/middleware"
+)
+
+func Test(t *testing.T) {
```

Comment:
Mind if we name this something like TestGzipHandler just for the benefit of using `go test -v`?